pkg/manager/manager.go

```diff
@@ -62,6 +62,9 @@ func NewManager(config *config.Config) (*Manager, error) {
 		if err != nil {
 			return nil, err
 		}
+		if w.GetConfig()["disabled"] == true {
```

Comment:
Should this happen before NewWorker as we want to completely skip it?

Reply:
Yes, but it seems that we could not get config without creating worker.